app/uk/gov/hmrc/trackingconsentfrontend/assets/src/domain/userPreferencesFactory.ts

```diff
@@ -1,99 +1,99 @@
-      { sameSite: 'strict', expires: 3650 }
-    )
-  }
-
-  const userAcceptsAll = () => {
-    storePreferences({
-      acceptAll: true
-    })
-    sendPreferences(CONSENT_UPDATED_EVENT)
-  }
-
-  const setPreferences = (preferencesIn: Preferences) => {
-    storePreferences({
-      measurement: preferencesIn.measurement,
-      marketing: preferencesIn.marketing,
-      settings: preferencesIn.settings
-    })
-    sendPreferences(CONSENT_UPDATED_EVENT)
-  }
-
-  const isPreferenceStrictlyBoolean = ([, value]) => typeof value === 'boolean'
+      { sameSite: 'strict', expires: 3650 },
+    );
+  };
+
+  const isPreferenceStrictlyBoolean = ([, value]) => typeof value === 'boolean';
 
   const getSanitisedPreferences = (preferences: any) => {
     const entries: [string, boolean][] = cookieTypes.map((cookieType: string) => {
-      const { [cookieType]: value } = preferences
+      const { [cookieType]: value } = preferences;
 
-      return [cookieType, value]
-    }, [])
+      return [cookieType, value];
+    }, []);
 
-    return entries.filter(isPreferenceStrictlyBoolean)
-  }
+    return entries.filter(isPreferenceStrictlyBoolean);
+  };
 
-  const allThePreferences = () => fromEntries(cookieTypes.map((cookieType: string) => [cookieType, true]))
+  const allThePreferences = () => fromEntries(cookieTypes.map((cookieType: string) => [
+    cookieType,
+    true,
+  ]));
 
   const getPreferences = (): Preferences | undefined => {
-    const cookie = Cookies.getJSON(COOKIE_CONSENT)
+    const cookie = Cookies.getJSON(COOKIE_CONSENT);
     if (cookie === null || cookie === undefined || cookie.preferences === undefined) {
-      return undefined
+      return undefined;
     }
 
-    const { version } = cookie
+    const { version } = cookie;
     if (version !== COOKIE_VERSION) {
-      return undefined
+      return undefined;
     }
 
-    const { preferences } = cookie
+    const { preferences } = cookie;
     if (preferences.acceptAll === true) {
-      return allThePreferences()
+      return allThePreferences();
     }
 
-    const sanitisedPreferences = getSanitisedPreferences(preferences)
-
-    return sanitisedPreferences.length > 0 ? fromEntries(sanitisedPreferences) : undefined
-  }
+    const sanitisedPreferences = getSanitisedPreferences(preferences);
 
-  const getUserHasSavedCookiePreferences = () => getPreferences() !== undefined
+    return sanitisedPreferences.length > 0 ? fromEntries(sanitisedPreferences) : undefined;
+  };
 
-  const sendPreferences = (event: string) => {
-    subscribers.forEach((subscriber: Communicator) => subscriber.sendPreferences(self, event))
-  }
+  const getUserHasSavedCookiePreferences = () => getPreferences() !== undefined;
 
   const self = {
-    userAcceptsAll,
-    setPreferences,
     getUserHasSavedCookiePreferences,
     getPreferences,
-    sendPreferences,
-    subscribe
-  }
-
-  return self
-}
-
-export default userPreferencesFactory
+    subscribe,
+    userAcceptsAll: () => {
```

Comment:
For me having all of the functions inline like this now does make it harder to read. Is it possible to keep eslint happy while retaining the previous structure? I know we've been over this before but this feels like a backwards step.

Reply:
The problem here is that it's particular about the order lines of code appear - the code was never a problem here but it was a challenge to persuade eslint that it wasn't a problem :)

If you'd like to suggest an alternative I'd be happy to check it out.

Reply:
I'm not sure you liked my previous suggestions :-) Should we just ignore this rule then with a comment?

Reply:
I'm happy to do that.